tests/apps/pubsub-bulk-subscriber/app.go

```diff
@@ -115,6 +116,7 @@ type AppBulkMessageEntry struct {
 type BulkSubscribeResponseEntry struct {
 	EntryId string `json:"entryId"` //nolint:stylecheck
 	Status  string `json:"status"`
+	Source  string `json:"strouce"`
```

Comment:
Typo

Reply:
Thanks!

Reply:
Still looks not fixed to me, maybe this is a reason for the test failures?